config/sync/views.view.authors.yml

```diff
@@ -33,7 +33,7 @@ display:
       query:
         type: views_query
         options:
-          disable_sql_rewrite: false
+          disable_sql_rewrite: true
```

Comment:
What is this change and what does it do?

Reply:
it's the thing that we decided to match across both author fields. It's unclear what exactly it does because it didn't affect anything.

Reply:
"it's the thing that we decided to match across both author fields." no idea what you're talking about, can i have some more information please and also figure out what it does please.

Reply:
sql rewrite can sometimes block access to content for certain users, so that's why it's better to have it disabled in this case

Reply:
Okay so that's what it does, how did you know to disable this when you didn't know what it did till 3 minutes ago?

Reply:
Also which certain users does it block access for? and in what situations?

Reply:
I didn't know what it did because I didn't change it myself. I think it happened when I added the views reference module and fiddled with the field. I'm not 100% sure though. It blocks access to anonymous users in my case.

Reply:
Okay, so is the view still viewable by logged out users?

Reply:
yeah